certified-connectors/Calendly/apiDefinition.swagger.json

```diff
@@ -767,10 +1349,6 @@
         "provider": {
           "type": "string",
           "description": "Payment provider.",
-          "enum": [
```

Comment:
The list of enum values has been removed from the input field. Existing flows may have this value chosen and will continue to send it.
can you please confirm that no users will be affected by this change?